docs/applications/deploy-application.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Deploy application
+intro:
+links:
+    overview:
+    quickstart:
+    previous:
+        - /docs/configuration-files/create-dotenv-file
```

Comment:
1) I see you're using the terms `add` and `create` interchangeably. Can we stick to `add` as this is the terminology adopted on Devopness web app? 

2) `DotEnv` is only one of the multiple file formats that can be managed via Devopness. Beyond `.env` users can add `.yaml`, `.txt`, `.md`, `.<anything>`.

I believe the documentation topics should be agnostic and re-usable across a multitude of data variation. If any, `DotEnv` can be mentioned in the configuration files steps as an example, but I don't think having a dedicated documentation topic just for `.env` files is needed:
```suggestion
        - /docs/configuration-files/add-configuration-file
```